tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,238 @@
-# already defined ``orig_time`` of annotations caused its ``onset`` to be
-# recalibrated with respect to the new ``orig_time``. As a result both
-# annotations have now identical ``onset`` and identical ``orig_time``:
-
-# Show the annotations in the raw objects
-print(raw_a.annotations)
-print(raw_b.annotations)
-
-# Show that the onsets are the same
-np.set_printoptions(precision=6)
-print(raw_a.annotations.onset)
-print(raw_b.annotations.onset)
-
-###############################################################################
-# Notice again that for the case where ``orig_time`` is ``None``,
-# it is assumed that the ``orig_time`` is the time of the first sample of data.
-
-raw_delta = (1 / raw.info['sfreq'])
-print('raw.first_sample is {}'.format(raw.first_samp * raw_delta))
-print('annot_none.onset[0] is {}'.format(annot_none.onset[0]))
-print('raw_a.annotations.onset[0] is {}'.format(raw_a.annotations.onset[0]))
-
-###############################################################################
-# Valid operations in :class:`mne.Annotations`
-# --------------------------------------------
+# The Events and Annotations data structures
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Concatenate
-# ~~~~~~~~~~~
+# Generally speaking, both the Events and :class:`~mne.Annotations` data
+# structures serve the same purpose: they provide a mapping between times
+# during an EEG/MEG recording and a description of what happened at those
+# times. In other words, they associate a *when* with a *what*. The main
+# differences are:
 #
-# It is possible to concatenate two annotations with the + operator (just like
-# lists) if both share the same ``orig_time``
-
-annot = mne.Annotations(onset=[10], duration=[0.5],
-                        description=['foobar'],
-                        orig_time=orig_time)
-annot = annot_orig + annot  # concatenation
-print(annot)
-
-###############################################################################
-# Iterating, Indexing and Slicing :class:`mne.Annotations`
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 1. the Events data structure represents the *when* in terms of samples,
+#    whereas the :class:`~mne.Annotations` data structure represents the *when*
+#    in seconds.
+# 2. the Events data structure represents the *what* as an integer "Event ID"
+#    code, whereas the :class:`~mne.Annotations` data structure represents the
+#    *what* as a string.
+# 3. Events in an Event array do not have a duration (though it is possible to
+#    represent duration with pairs of onset/offset events within an Events
+#    array), whereas each element of an :class:`~mne.Annotations` object
+#    necessarily includes a duration (though the duration can be zero if an
+#    instantaneous event is desired).
+# 4. Events are stored as an ordinary :class:`NumPy array <numpy.ndarray>`,
+#    whereas :class:`~mne.Annotations` is a :class:`list`-like class defined in
```

Comment:
+1 for adding a short title to each bullet.  The reason I said it's list-like is because it has an `append` method. If you think it's confusing I can remove it; I would not say "quacks like a list".